src/components/app/SingleStrategy/GenLender.tsx

```diff
@@ -57,7 +57,11 @@ export const GenLender = (props: GenLenderProps) => {
     };
 
     const renderData = () => {
-        const lenderStatuses = data ? (
+        if (!data) {
+            return <></>;
```

Comment:
Much better! Thanks @edwinzhng

nit: Usually the React team recommends returning `null` explicitly rather than Fragment, it has some optimisation benefits.

Reply:
Ah good to know, I'll fix when I tackle the general cleanup